Classes/Http/RestRequestHandler.php

```diff
@@ -73,7 +74,11 @@ public function handleRequest(\Psr\Http\Message\ServerRequestInterface $request)
 
         // Dispatch the API-call
         $objectManager = GeneralUtility::makeInstance(ObjectManager::class);
-        $objectManager->get(Dispatcher::class)->dispatch();
+        if (interface_exists('\Psr\Http\Server\MiddlewareInterface')) {
+            $objectManager->get(Dispatcher::class)->dispatch();
```

Comment:
There is no dispatch() method any more in the dispatcher class, right?

Reply:
That code path was unused, as the RestRequestHandler is not registered when we have a MiddlewareInterface. I removed that if/else block. Thanks for noticing.